app/utils/DateUtils.scala

```diff
@@ -16,33 +16,68 @@
 
 package utils
 
-import org.joda.time.{DateTime, LocalDate}
-import play.api.libs.json.{Format, JodaReads, JodaWrites}
+import play.api.libs.json._
+
+import java.time.format.DateTimeFormatter
+import java.time.{LocalDate, OffsetDateTime, ZonedDateTime}
+import scala.util.{Failure, Success, Try}
 
 object DateUtils {
```

Comment:
Is it worth having some tests for these formatters and read and writes? They're pretty fundamental to the service?